gfx/portraits/portrait_modifiers/01_clothes.txt

```diff
@@ -52,6 +171,10 @@ clothes = {
 		}
 		weight = {
 			base = 0
+			modifier = {
+				add = 2000
+				this = character:33922
+			}
```

Comment:
We don't have a character with the same id in the mod, but it would be bad if we will.

Reply:
Purged in 436c093c2b5da4795ec78f91b4e362d17c75493d.